src/controller_model/distributionList.ts

```diff
@@ -30,6 +30,7 @@ export class DistributionListControllerModel implements threema.ControllerModel<
     public name: string;
     public subject: string;
     public isLoading = false;
+    public readonly requiredMemberFeatureMask = null; // No constraints
```

Comment:
Do you mean `any` by that? But it clearly isn't `any`.

Reply:
I can't follow. What do you mean?